src/ccl/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Climate Change Lab
+ * Sidebar component
+ */
+import { Component, ViewEncapsulation, EventEmitter, Input, Output } from '@angular/core';
+
+import { IndicatorsService } from '../services/indicators.service';
+import { ChartService } from '../services/chart.service';
+
+/*
+ * Sidebar Component
+ * Populates sidebar with indicators and triggers adding charts
+ */
+
+@Component({
+  selector: 'sidebar',
+  encapsulation: ViewEncapsulation.None,
+  providers: [IndicatorsService, ChartService],
+  styleUrls: [
+      '../../assets/css/sidebar.css',
+  ],
+  templateUrl: './sidebar.template.html'
+})
+export class Sidebar {
+    private indicatorsList: string;
+
+    // Pull data to populate sidebar
+    constructor(private chartService: ChartService, private indicatorService: IndicatorsService){};
+
+    // Set up click event handlers
+    onIndicatorClicked(indicator) {
+        this.chartService.addChart(indicator);
+    }
+
+    ngOnInit() {
```

Comment:
Indeed. Just helpful type annotations then to make it easier to see what the class is doing. Also provides type checking in the case of you misspelling the method name or something and getting confused as to why the method isn't called.